examples/todo/cpp/main.cpp

```diff
@@ -44,5 +44,26 @@ int main()
         return slint::CloseRequestResponse::HideWindow;
     });
 
+    demo->set_is_header_visible(true);
+
+    demo->on_sort_filter([todo_model, demo = slint::ComponentWeakHandle(demo)] {
+        (*demo.lock())->set_todo_model(todo_model);
```

Comment:
maybe you'd want to store demo.lock() in a local variable to make the code a bit more concise here